app/models/item.rb

```diff
@@ -5,4 +5,17 @@ class Item < ApplicationRecord
 
   validates_presence_of :name, :category
   validates_numericality_of :count, greater_than: 0
+
+  enum status:{
+    "Tents" => 1,
```

Comment:
@sandisz-d734m37 I think you want this to be `category` instead of `status` because that is what the attribute is.

Reply:
Tha

> @sandisz-d734m37 I think you want this to be `category` instead of `status` because that is what the attribute is.

That was my fault, I told him wrong code. We can get that fixed. I'll close the request and fix it on my end for it to be pulled through.